tests/utils.ts

```diff
@@ -17,6 +17,69 @@ export enum TableTypes {
   AllTypes,
 }
 
+type TablePlotNames =
```

Comment:
Split this into `TableNames`, `PlotNames`, and then `type ObjectNames = TableNames | PlotNames`